index.html

```diff
@@ -0,0 +1,91 @@
+    <title>Product Landing Page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+<header id = "header">
+    <img src = "/images/company-logo.png" alt = "Company Logo" id = "header-img">
+    <nav id = "nav-bar">
+          <button class = "nav-link" href = "#Features">Features</button>
+          <button class = "nav-link" href = "#How_It_Works">How It Works</button>
+          <button class = "nav-link" href = "#Pricing">Pricing</button>
+    </nav>
+   
+   
+    
+</header>
+<div class="div-class">
+  <h1>Prototype Title</h1>
+<video id = "video" controls width = "250">
+<source src = "images/video.mp4" type="video/mp4">
+</video>
+<form id = "form" method="POST" action="https://www.freecodecamp.com/email-submit">
+    <br>
+    <br>
+    <div id = "div-input">
+<input type = "email" id = "email" placeholder="Insert e-mail here..." name = "email">
+<input id="submit" type="submit" value="submit">
+</div>
+<br>
+<br>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    
+    <script  src="js/index.js"></script>
+    <section id = "Features" class = "nav-link">
+    <h2>Features</h2>
+    <h3>Premium Materials</h3>
+    <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase. </p>
+    <h3>Fast Shipping</h3>
+    <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied. </p>
+    <h3>Quality Assurance</h3>
+    <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument. </p>
+    <p></p>
+    </section>
+    <section id = "How_It_Works">
+      <h2>How It Works</h2>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+      <p></p>
+      </section>
+    <section id = "Pricing">
+    <h2>Pricing</h2>
+    </section>
+    <div class="flex-container">
+    <div id="bronze">
+      <h2>Bronze Package</h1>
+        <ul>
+          <li>Lorem Ipsum</li>
+          <li>Lorem Ipsum</li>
+          <li>Lorem Ipsum</li>
+        </ul>
+
+    </div>
+    <div id="gold">
+      <h2>Gold Package</h1>
+        <ul>
+          <li>Lorem Ipsum</li>
+          <li>Lorem Ipsum</li>
+          <li>Lorem Ipsum</li>
+        </ul>
+
+    </div>
+    <div>
+      <h2 id="platinum">Platinum Package</h1>
+        <ul>
+          <li>Lorem Ipsum</li>
+          <li>Lorem Ipsum</li>
+          <li>Lorem Ipsum</li>
+        </ul>
+
+    </div>
+  </div>
+   
+
+
+  </div>
+  </body>
+
```

Comment:
To make your code look cleaner remove the whitespace between the body closing tag and HTML closing tag.

Reply:
- [ ] I suggest you to phrase comments kindly requesting changes:
✔️ * This is a polite suggestion: *"I think it would be better to make README file more descriptive"*
🚫 * This reads as an order: *"README file should be a lot more descriptive"*